src/WebJobs.Script/Scale/FunctionsScaleManager.cs

```diff
@@ -39,13 +57,28 @@ public FunctionsScaleManager(IScaleMonitorManager monitorManager, IScaleMetricsR
         /// <returns>The scale vote.</returns>
         public virtual async Task<ScaleStatusResult> GetScaleStatusAsync(ScaleStatusContext context)
         {
-            var monitors = _monitorManager.GetMonitors();
+            GetScaleStatusProviders(_monitorManager, _targetScalerManager, out IEnumerable<IScaleMonitor> scaleMonitors, out IEnumerable<ITargetScaler> targetScalers);
```

Comment:
This GetScaleStatusProviders method isn't providing any value. Let's remove it and just pass in the managers to GetScalersToSample and have it do everything internally. Also, move the utility method GetScalersToSample from Utility class to FunctionsScaleManager please.